website/templates/profile/notifications.mako

```diff
@@ -48,13 +47,16 @@
             </div>
     </div>
 </div>
+</%def>
 
-<script type="text/javascript">
-
-    $script(['/static/js/notificationsConfig.js']);
-    $script.ready('NotificationsConfig', function() {
-        var notifications = new NotificationsConfig('#selectLists', '${website.settings.MAILCHIMP_GENERAL_LIST}');
-    });
-</script>
-
+<%def name="javascript()">
+    <% import website%>
+    <script type="text/javascript">
+        window.contextVars = $.extend({}, window.contextVars, {'notificationsSelector': '#selectLists',
```

Comment:
We use `contextVars` when we need to use Mako variables in  a JS module. Since the `#selectLists` ID does not vary, I don't think it needs to be pt in `contextVars`.